cli/index.ts

```diff
@@ -11,7 +11,7 @@ import { dirname, join, resolve, extname as getExtName } from 'path';
 import * as zlib from 'zlib';
 
 // @ts-ignore
-import { createStore, loadAndBundleSpec, Redoc } from 'redoc';
+import { createStore, loadAndBundleSpec, Redoc } from 'redoc-etn';
```

Comment:
note: i should have created a separate branch for these changes, but I made them on the same branch for the PR. This change was just so that I could run an `npm install -g redoc-cli-etn` that would use the version of redoc that includes `x-badges` support